### PR TITLE
Rename market item currency

### PR DESCRIPTION
**Pull request by Valansch**
You don't pay with market items, you pay with currency.
You don't go to the supermarket and pay your supermarket_things with supermarket_thing.

### CODE REVIEW

**Comment by Valansch** [comment]
dependent on and waiting for #634

File: features/market.lua [hunk 1]
```diff
@@ -224,17 +226,21 @@ local function player_created(event)
...
+        admin_only = true,
+    },
+    spawn_market
+)
 
 Event.on_nth_tick(180, on_180_ticks)
 Event.add(defines.events.on_pre_player_mined_item, pre_player_mined_item)
 Event.add(defines.events.on_entity_died, fish_drop_entity_died)
-Event.add(Retailer.events.on_market_purchase, on_market_purchase)
+Event.add(defines.events.on_market_item_purchased, market_item_purchased)
```
Comment (linaori) [inline features/market.lua]: You break it, you buy it

**Review by linaori — CHANGES_REQUESTED** [review]

File: features/market.lua (hunk 1 above)
Comment (Valansch) [reply features/market.lua]: Ehh, that was broken in #634 by my merge. Couldn't figure out what was different. Oops!

**Review by linaori — APPROVED** [review]

**Comment by mheguy** [comment]
Looks good to me, merging with my thanks.

**Pull request merged**